package.json

```
@@ -40,6 +40,7 @@
     "boilerplate"
   ],
   "dependencies": {
+    "bcrypt": "false3.0.6",
```

```suggestion
    "bcrypt": "3.0.6",
```
and run `yarn config set save-prefix ""` so that yarn won't add `false` before version numbers in the future